e2e_samples/parking_sensors/scripts/deploy_infrastructure.sh

```diff
@@ -175,11 +175,29 @@ appinsights_connstr=$(az monitor app-insights component show \
 az keyvault secret set --vault-name "$kv_name" --name "applicationInsightsKey" --value "$appinsights_key"
 az keyvault secret set --vault-name "$kv_name" --name "applicationInsightsConnectionString" --value "$appinsights_connstr"
 
+# ###########################
+echo "validate_password function..."
+# ###########################
+###if there is an hyphen in the first position replace by R
+# ###########################
+validate_password() {
```

Comment:
as discussed I doubt this will work, suggest you instead ask for another secret if the password contains an hyphen in the first char.